mediapipe/tasks/ios/vision/core/sources/MPPVisionTaskRunner.mm

```diff
@@ -175,81 +179,112 @@ - (nullable instancetype)initWithTaskInfo:(MPPTaskInfo *)taskInfo
+        [MPPVisionPacketCreator createPacketWithNormalizedRect:rect.value()];
+
+    inputPacketMap[_normRectInputStreamName.cppString] = normalizedRectPacket;
+  }
+
   Packet imagePacket = [MPPVisionPacketCreator createPacketWithMPPImage:image error:error];
   if (imagePacket.IsEmpty()) {
     return std::nullopt;
   }
 
-  Packet normalizedRectPacket =
-      [MPPVisionPacketCreator createPacketWithNormalizedRect:rect.value()];
+  inputPacketMap[_imageInputStreamName.cppString] = imagePacket;
 
-  PacketMap inputPacketMap = InputPacketMap(imagePacket, normalizedRectPacket);
   return inputPacketMap;
 }
 
 - (std::optional<PacketMap>)inputPacketMapWithMPPImage:(MPPImage *)image
                                       regionOfInterest:(CGRect)roi
                                timestampInMilliseconds:(NSInteger)timestampInMilliseconds
                                                  error:(NSError **)error {
-  std::optional<NormalizedRect> rect =
-      [self normalizedRectWithRegionOfInterest:roi
-                                     imageSize:CGSizeMake(image.width, image.height)
-                              imageOrientation:image.orientation
-                                         error:error];
-  if (!rect.has_value()) {
+  if (![self shouldTaskContinueWithRegionOfInterest:roi
+                                   imageOrientation:image.orientation
+                                              error:error]) {
     return std::nullopt;
   }
 
+  PacketMap inputPacketMap;
+
+  if (_normRectInputStreamName) {
+    std::optional<NormalizedRect> rect =
+        [self normalizedRectWithRegionOfInterest:roi
+                                       imageSize:CGSizeMake(image.width, image.height)
+                                imageOrientation:image.orientation
+                                           error:error];
+    if (!rect.has_value()) {
+      return std::nullopt;
+    }
+
+    Packet normalizedRectPacket =
+        [MPPVisionPacketCreator createPacketWithNormalizedRect:rect.value()
+                                       timestampInMilliseconds:timestampInMilliseconds];
+
+    inputPacketMap[_normRectInputStreamName.cppString] = normalizedRectPacket;
+  }
+
   Packet imagePacket = [MPPVisionPacketCreator createPacketWithMPPImage:image
                                                 timestampInMilliseconds:timestampInMilliseconds
                                                                   error:error];
   if (imagePacket.IsEmpty()) {
     return std::nullopt;
   }
 
-  Packet normalizedRectPacket =
-      [MPPVisionPacketCreator createPacketWithNormalizedRect:rect.value()
-                                     timestampInMilliseconds:timestampInMilliseconds];
+  inputPacketMap[_imageInputStreamName.cppString] = imagePacket;
 
-  PacketMap inputPacketMap = InputPacketMap(imagePacket, normalizedRectPacket);
   return inputPacketMap;
 }
 
-- (std::optional<PacketMap>)processImage:(MPPImage *)image
```

Comment:
This showed up during the import:

```

method definition for 'processImage:regionOfInterest:error:' not found [-Werror,-Wincomplete-implementation]
   54 | @implementation MPPVisionTaskRunner
      |                 ^
./third_party/mediapipe/tasks/ios/vision/core/sources/MPPVisionTaskRunner.h:98:1: note: method 'processImage:regionOfInterest:error:' declared here
   98 | - (std::optional<mediapipe::tasks::core::PacketMap>)processImage:(MPPImage *)image
[More output omitted...]
``` 

Can you look at this?

Reply:
Sorry. Fixed it. I accidentally deleted a method while committing.